zookeeper-contrib/zookeeper-contrib-loggraph/src/main/java/org/apache/zookeeper/graph/Log4JSource.java

```diff
@@ -115,20 +115,16 @@ public Log4JSourceIterator(Log4JSource src, long starttime, long endtime, Filter
 	}
 	
 	synchronized public long size() throws IOException {
-	    if (LOG.isTraceEnabled()) {
-		LOG.trace("size() called");
-	    }
+	  LOG.trace("size() called");
```

Comment:
Argh!  Some of these files are using (8-)tabs for indentation…  Still, the indentation of this statement (and a few others) is inconsistent.